indico/web/client/styles/themes/indico.scss

```diff
@@ -53,6 +53,7 @@ $selected-color: $indico-blue !default;
 
 body {
   background: #424242;
+  overflow-x: scroll;
```

Comment:
I'm pretty sure this results in a permanent horizontal scrollbar. What about `auto` to show it only if one is needed?

Reply:
I just checked  again, It does not end up creating permanent horizontal scrollbar but I think `auto` would be enough since both `scroll` and `auto` produce same results on my local instance.